### PR TITLE
service catalog: create service and ssl certs for controller manager

### PR DESCRIPTION
**Pull request by jboyd01**
create k8s service and ssl certs for controller manager
change controller manager port & annotate it as secure
add Prometheus scrape for Catalog controller manager over ssl
change log levels for both API Server and Controller Manager to -v 3

### CODE REVIEW

**Comment by jboyd01** [comment]
@jpeeler would you please give this a quick review?

File: roles/openshift_service_catalog/tasks/generate_certs.yml [hunk 1]
```diff
@@ -35,6 +45,15 @@
     signer_key: "{{ generated_certs_dir }}/ca.key"
     signer_serial: "{{ generated_certs_dir }}/apiserver.serial.txt"
 
+- name: Generating Controller Manager keys
```
Comment (smarterclayton) [inline roles/openshift_service_catalog/tasks/generate_certs.yml]: Why aren't we using service serving certs here?  That's what every component should be using to listen on TLS.

Comment (jboyd01) [reply roles/openshift_service_catalog/tasks/generate_certs.yml]: @smarterclayton I'm not certain why this was done this way originally, I followed what was in place for the catalog API Server.  I can create a new item to cut both of these over to use service serving certs in the next release, or must this be changed for 3.10?

**Pull request merged**